ui/tutorial-renderer/src/main/res/values/tutorial_style.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="Theme.GodTools.AppBar.Tutorial" parent="Theme.AppCompat.Light">
+        <item name="android:background">@android:color/transparent</item>
+        <item name="android:tint">@color/gt_blue</item>
+        <item name="actionMenuTextColor">@color/gt_blue</item>
+    </style>
```

Comment:
you are combining a style and a theme together which are 2 distinct things

Reply:
@frett it looks like you took care of this?